### PR TITLE
OBSDOCS-406 - Logging 5.6.9 & 5.5.14 RNs

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.10, 4.11, 4.12

Issue: [OBSDOCS-406](https://issues.redhat.com//browse/OBSDOCS-406)

Link to docs preview: https://drive.google.com/file/d/1NRFnGZrAvCoC0Zg4SgnQwLA2LDr7wMOL/view?usp=sharing
Note: you must download the file, then open it to view as rendered HTML. 'Preview' will show raw HTML.

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by kabirbhartiRH** [comment]
I think https://issues.redhat.com/browse/LOG-4357 needs to be removed from 5.5.14 as it is NEC restricted ticket.

Otherwise lgtm

**Review by anpingli — COMMENTED** [review]
LGTM

File: modules/logging-rn-5.6.9.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+// cluster-logging-release-notes.adoc
+// logging-5-6-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-9_{context}"]
+= Logging 5.6.9
+This release includes link:https://access.redhat.com/errata/RHBA-2023:4342[OpenShift Logging Bug Fix Release 5.6.9].
+
+[id="openshift-logging-5-6-9-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, When multiple roles were used to authenticate via STS with AWS Cloudwatch forwarding, a recent update caused the credentials to be non-unique. With this update, multiple combinations of STS roles and static credentials can once again be used to authenticate with AWS Cloudwatch. (link:https://issues.redhat.com/browse/LOG-4084[LOG-4084])
```
Comment (mburke5678) [inline modules/logging-rn-5.6.9.adoc:11]: s/When/when

**Comment by mburke5678** [comment]
A few nits. Otherwise LGTM

**Pull request merged**